templates/zerver/help/add-emoji.md

```diff
@@ -11,7 +11,6 @@ built-in emoji menu, pasting Unicode, or typing the emoji code.
 
```

Comment:
All links that go to `(/help/#posting-and-replying)` should be linked to the other **Replying to a message** doc we have since it might be removed.